lib/api.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
-var https       = require('https');
-var Promise     = require('bluebird');
+var https = require('https');
+var Promise = require('bluebird');
 
-var logger      = require('./logger');
-var pkgjson     = require('../package.json');
+var logger = require('./logger');
+var pkgjson = require('../package.json');
 
-var ApiError    = require('./errors/api-error.js');
```

Comment:
Can't we use this style? 😢

Reply:
Actually good point. Can we adjust rule so we can align assignment expression? @varshard

Reply:
I saw that some assignments are aligned in some and not aligned in some.
I will try to preserve the alignment, then.

Reply:
Let's force align all of them then 😄

Reply:
Unfortunately that there are no such option. 😢